package.json

```
@@ -17,7 +17,7 @@
     "test:coverage": "jest --coverage",
     "eslint": "eslint --ext .tsx,.js ./src/",
```

can you also fix this
```suggestion
    "eslint": "eslint --ext .tsx,.ts ./src/",
```

---

Good catch, updated